include/fmt/format-inl.h

```diff
@@ -1169,6 +1125,10 @@ template <> struct cache_accessor<double> {
   }
 };
 
+FMT_FUNC uint128_fallback get_cached_power(int k) noexcept {
+  return cache_accessor<double>::get_cached_power(k);
+}
```

Comment:
Do we need this function? It seems to be used only in one place and can be replaced with calling `cache_accessor<double>::get_cached_power` directly.

Reply:
The reason I did that way was I didn't want to move the entire `cache_accessor<double>` class to `format.h`. I guess we will anyway need to move the the table into `format.h` to `constexpr`-ify the current implementation, but I considered that as a job of a later work. Or is there a simpler solution?

Reply:
I see. Let's keep `get_cached_power` for now.